Composer/packages/client/src/components/Pagination/index.tsx

```diff
@@ -16,52 +17,52 @@ export interface IPaginationProps {
 }
 
 const createDropdownOption = (pageCount: number) => {
-  const options: IDropdownOption[] = [];
-  for (let i = 1; i <= pageCount; i++) {
-    options.push({ key: 'page' + i, text: '' + i });
-  }
-  return options;
+  return range(pageCount).map((_, i) => ({
+    key: `page ${i}`,
+    text: `${i}`,
+  }));
 };
 
 export const Pagination: React.FC<IPaginationProps> = (props) => {
-  const [index, setIndex] = useState(0);
+  const [index, setIndex] = useState<number>(0);
   const { pageCount, onChange } = props;
 
   useEffect(() => {
     setIndex(0);
   }, [pageCount]);
 
+  const changePage = (page: number) => {
+    setIndex(page);
+    onChange(page + 1);
+  };
+
   const handlePageSelected = (event: React.FormEvent<HTMLDivElement>, item?: IDropdownOption, index?: number) => {
-    setIndex(index || 0);
-    onChange(index ? index + 1 : 1);
+    if (index == null) return;
```

Comment:
Double `==` here. Use `===` instead.

Reply:
I think this is one place I feel comfortable carving out an exception. `== null` is currently the cleanest way there is to say "is `null` or `undefined`", and in TypeScript that's a common thing to say because of how it narrows a `?` type into a realized one.